cnxarchive/tests/views/test_exports.py

```diff
@@ -314,13 +314,32 @@ def test_exports(self):
         export = get_export(self.request).body
 
         self.assertEqual(self.request.response.content_disposition,
-                         "attached; filename=college-physics-{}.pdf"
+                         "attached; filename=college-physics-7.1.pdf;"
+                         " filename*=UTF-8''college-physics-7.1.pdf"
                          .format(version))
```

Comment:
After the change, this seems to be doing a `.format` on a string without any placeholder brackets?